zapier/introduction/zapier-filtering/zapier-what-are-filters.md

```diff
@@ -0,0 +1,37 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# What Are Filters?
+
+---
+## Content
+
+Filters represent a very important part of Zapier because they allow you to control which data goes forward in the flow. Basically, if the received information does not fit a certain constraint (defined by you), then the flow won't continue. 
+
+Adding a filter is as easy as adding a new step, all you have to do is choose `'Filter by Zapier'` as your app.
+
+We have briefly used a filter in the past to check whether a column in our spreadsheet had a certain value or not. One important feature of filters is that they are not limited to one constraint per step.
```

Comment:
```suggestion
We have briefly used a filter in the past to check whether a column in our spreadsheet had a certain value or not. Keep in mind that filters are not limited to one constraint per step.
```

also, would using "check" instead of "constraint" be clearer?